max_distance : 5
map_resolution : 0.025
dilation_kernel : 2


